collectors/bzimport/convertors.py

```diff
@@ -548,6 +548,45 @@ def flags(self):
 
         return [flag for flag in self.flaw_bug["flags"] if isinstance(flag, dict)]
 
+    @staticmethod
```

Comment:
I do not like using `@staticmethod` but I did not want to change how the `flags` property works, and I was not able to come up with a better solution. So I would appreciate any recommendation.

Reply:
Since `flags` is attribute of the `FlawCollector`, why not simply just turn this into the `@property` and instead of passing the flags as an argument, access them through the `self`.  In fact, this can be a cached_property, since it is pointless to run the parsing logic again when calling the property for the second time.

Reply:
Thanks! This is what I was looking for. I was not able to realize that I could access a property from another property :)